### DISCUSSION TITLE
Webpage won't load after changing URL in Advanced Settings

### DISCUSSION THREAD

**Discussion by bcardi0427** [mcguirepr89/BirdNET-Pi#763] [Q&A] [answered]
I tried setting a domain in the advanced settings that I have and it just got the webpage stuck loading only the top of the page. I changed it back to nothing in the url and then to it's IP address in the url and same thing happens.

**Comment by bcardi0427** [comment, marked as answer]
It looks like all the services are running and files are being processed so it may be something in writing to the html or PHP.

**Reply by ehpersonal38** [reply]
Hey there @bcardi0427!

I was able to recreate the issue you were experiencing and I have a solution that should get you up and running again in no time. :)

To fix this, you'll need to follow these steps:

1. Edit the file `~/BirdNET-Pi/birdnet.conf` to remove the trailing slash from your `BIRDNETPI_URL`. Here's an example to help you out:
Incorrect: `http://192.168.1.71/`
Correct: `http://192.168.1.71`
Alternatively, you can just leave the URL blank.

2. After making changes to your configuration file, you'll need to let the webserver know by running this command: 
`sudo bash ~/BirdNET-Pi/scripts/update_caddyfile.sh`

That's it! Your BirdNET-Pi should be back up and running. Let me know if you have any questions or need further assistance.

And thanks @lloydbayley for pointing out the little tidbit about trailing slashes messing up the webserver, had I not remembered it I would have no idea what went wrong here.

**Reply by bcardi0427** [reply]
It also picked up everything and only the webpage wasn't working! All is fine now as you can see from the link.

http://birdnetpi.bcardi.org/

**Comment by bcardi0427** [comment]
That did it, thanks a bunch!!
